bin/update_or_create_webhooks.py

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+import json
+
+import click
+from openedx_webhooks.lib.github.utils import update_or_create_webhook
+
+click.disable_unicode_literals_warning = True
+
+
+@click.command()
+@click.argument('config-json', type=click.File('rb'))
+def cli(config_json):
+    """
+    Update or create the specified webhook into repos using a
+    configuration JSON file:
+
+    \b
```

Comment:
I don't understand where this JSON file is coming from? We already had repos.json, and that was replaced with openedx.yml files in the repos themselves.  Can't we use the openedx.yml files instead of creating a new configuration file?